ci/infra/testrunner/tests/test_node_reboot.py

```diff
@@ -0,0 +1,37 @@
+import logging
+import time
+
+logger = logging.getLogger("testrunner")
+
+
+def test_reboot_master(deployment, platform, skuba, kubectl):
+    # Make sure there is at least two masters
+    assert skuba.num_of_nodes('master') > 1
+
+    logger.info('Wait for all the nodes to be ready')
+    kubectl.run_kubectl('wait --for=condition=Ready node --all --timeout=5m')
+
+    logger.info('Rebooting master 1')
+    platform.ssh_run('master', 1, 'sudo reboot')
+
+    # Give the reboot a bit to start
+    time.sleep(5)
+
+    kubectl.run_kubectl('wait --for=condition=Ready node --all --timeout=5m')
```

Comment:
This condition does not verify anything, it takes some time until node becomes `NotReady`.
```bash
# ssh sles@10.86.2.99 'sudo reboot'

# kubectl wait --for=condition=Ready node --all
node/caasp-master-kravciak-0 condition met
node/caasp-worker-kravciak-0 condition met # rebooted

# ssh 10.86.2.99
ssh: connect to host 10.86.2.99 port 22: Connection refused
```

Reply:
Yeah I haven't had a chance to run the test yet just ran the steps locally. That's why this is labeled wip

Reply:
Also be aware that the `--all` flag doesn't work yet as we're still on kubectl 1.13.2 on the jenkins workers, but i'll try to redeploy them today